library/src/main/java/io/constructor/data/model/browse/BrowseRequestBody.kt

```diff
@@ -0,0 +1,13 @@
+package io.constructor.data.model.browse
+
+import com.squareup.moshi.Json
+import io.constructor.data.model.common.*;
+import java.io.Serializable
+
+data class BrowseRequestBody(
```

Comment:
The naming here could be improved ... maybe browse result click body?  Also can we have one for browse result click and one for browse result load?